src/Core/Module/HookRepository.php

```diff
@@ -60,14 +60,29 @@ public function getIdByName($hook_name)
         return (int) $id_hook;
     }
 
+    /**
+     * Creates a new hook if not already existing and returns the hook id.
+     *
+     * @param string $hook_name The name of the hook
+     * @param string $title The title for the hook
+     * @param string $description The description for the hook
+     * @param int $position if the modules in the hook can be positioned
+     *
+     * @return int Hook ID
+     */
     public function createHook($hook_name, $title = '', $description = '', $position = 1)
     {
+        $id_hook = (int) $this->getIdByName($hook_name);
```

Comment:
```suggestion
        $id_hook = $this->getIdByName($hook_name);
```
It always return an int ^^

Reply:
Yes that true, just a habbit, (but not a bad one) :)